lib/perl/Genome/InstrumentData/Composite/Workflow.pm

```diff
@@ -110,9 +106,8 @@ sub _run_workflow {
     Genome::Sys->disconnect_default_handles;
 
     $self->debug_message('Running workflow...');
-    my $dag = Genome::WorkflowBuilder::DAG->from_xml($workflow->save_to_xml());
     my %inputs_hash = (@$inputs); # For some reason $inputs is an array-ref
```

Comment:
Good point; I'll make a follow-up PR once this one is done.  I think it'll be easier to follow as a fresh request rather than tacked on here.